barman/infofile.py

```diff
@@ -877,3 +877,22 @@ def get_child_backup_info(self, child_backup_id):
                     return backup_info
 
         return None
+
+    def is_full_and_eligible_for_incremental(self):
+        """
+        Used to filter out backups that have a parent backup id and are not considered
+        as FULL backups.
+
+        :note: Only consider backups which are eligible for Postgres core incremental
+        backups
+        backup_method = 'postgres', summarize_wal = 'on', parent_backup_id = None
```

Comment:
```suggestion
        .. note::
            Only consider backups which are eligible for Postgres core incremental
            backups:

            * backup_method = ``postgres``
            * summarize_wal = ``on``
            * parent_backup_id = ``None``
```

Adjusting to use the sphinx syntax for notes (`.. note::`). Also, using double backticks for literals.

The usage of a list is only a side suggestion.